test/unit/region.spec.js

```diff
@@ -599,6 +599,7 @@ describe('region', function() {
 
         render: function() {
```

Comment:
wow.. this test was just bad to begin with..
I think this should be
```js
template: _.template('<div class="sub-region"></div><div>some content</div>'),
onRender:
```
then no need to override the `render` function..
sorry you had to mess with that 😞